### PR TITLE
feat: make kafka maxPollRecords and maxPollIntervalMs configurable

### PR DESCRIPTION
**Pull request by elisamozammel**
make kafka maxPollRecords and maxPollIntervalMs configurable

https://louhos.atlassian.net/browse/VNOC-11083

### CODE REVIEW

File: src/main/java/io/zeebe/monitor/config/KafkaConfig.java [hunk 1]
```diff
@@ -27,6 +27,10 @@ public class KafkaConfig {
     public int numOfThreads;
     @Value("${kafka.streams.maxMessageSizeBytes:5242880}")
     private int maxMessageSizeBytes;
+    @Value("${kafka.streams.maxPollRecords:30}")
+    public int maxPollRecords;
+    @Value("${kafka.streams.maxPollIntervalMs:600000}")
+    public int maxPollIntervalMs;
```
Comment (Ninjaluo) [inline src/main/java/io/zeebe/monitor/config/KafkaConfig.java:33]: Should these defaults be the currently used values?

Comment (elisamozammel) [reply src/main/java/io/zeebe/monitor/config/KafkaConfig.java:33]: this is intended value. if it works fine, we don't have to change anything on chart

maxPollRecords default value is 500 which is huge for our case. 25 is used in process-manager. Based on that put the default value 30. If it works fine then we don't have to change anything on chart at this moment

File: src/main/java/io/zeebe/monitor/config/KafkaConfig.java [hunk 2]
```diff
@@ -81,7 +91,9 @@ public KafkaStreamsConfiguration streamsConfig() {
         props.put("num.stream.threads", this.getNumOfThreads());
         props.put("max.request.size", this.getMaxMessageSizeBytes());
         props.put("max.partition.fetch.bytes", (int)((double)this.getMaxMessageSizeBytes() * 1.1));
+        props.put("max.poll.interval.ms", this.getMaxPollRecords());
+        props.put("max.poll.records", this.getMaxPollIntervalMs());
```
Comment (Ninjaluo) [inline src/main/java/io/zeebe/monitor/config/KafkaConfig.java]: Seems the property assignments are swapped

Comment (elisamozammel) [reply src/main/java/io/zeebe/monitor/config/KafkaConfig.java]: my bad, good catch

**Review by Ninjaluo — APPROVED** [review]

**Pull request merged**